torch/csrc/jit/codegen/cuda/predicate_compute.cpp

```diff
@@ -219,6 +219,13 @@ kir::Bool* PredicateCompute::getInlinePredicate(
         return ir_builder.create<kir::Bool>(true);
       }
     }
+    // Never inline predicate block broadcasts
+    if (auto broadcast = dynamic_cast<const kir::BroadcastOp*>(expr)) {
+      const auto domain = broadcast->out()->as<kir::TensorView>()->domain();
+      if (domain->hasBlockBroadcast()) {
+        return ir_builder.create<kir::Bool>(true);
+      }
+    }
```

Comment:
We may want to rename `ignore_block_grid_external_ops` to something like `ignore_external_ops` or something.

Reply:
I went with `ignore_internal_syncthread_ops` and commented the function.